docs/ide/default-keyboard-shortcuts-in-visual-studio.md

```diff
@@ -80,6 +80,7 @@ The sections in the following table include commands that are global in that you
 |Debug.DOMExplorer|**Ctrl+Alt+V, D**|
 |Debug.EnableBreakpoint|**Ctrl+F9**|
 |Debug.Exceptions|**Ctrl+Alt+E**|
+|Debug.FunctionBreakpoint|**Ctrl+K, B**|
```

Comment:
```suggestion
|Debug.FunctionBreakpoint|**Ctrl+K, B** (Visual Studio 2019<br />**Ctrl**+**B** (Visual Studio 2017)|
```

Reply:
Oh yes I forgot about the VS2017/VS2019 thing. 

I guess then most of these changes should all be marked VS2019-only:

https://github.com/MicrosoftDocs/visualstudio-docs/commit/2a7d0f802aad96a5d819c0efb016735a41ac5bbf#diff-6c8142baa9ed932939b65d1b8e73b180

I might make another pull request to do that as well then...

As for this one:

In VS2017 Ctrl-K, B and Ctrl-K, Ctrl-B would both bring up the Code Snippets Manager.
Now in VS2018 Ctrl-K, B is function breakpoint (which is great) and Ctrl-K, Ctrl-B is still the snippets manager. That's OK I guess, but it think would be simpler if Ctrl-K, Ctrl-B and Ctrl-K, B both did the same thing as is the case with some other Ctrl-K chords. Snippet manager should move to some other key since it would be pretty uncommon for someone to ever bring that up - they would frankly never memorize a shortcut key for something so rarely used.

I guess the docs is not really the place to litigate that tho. :-)